docs/content/en/docs/reference/packagespec/metallb/v0.12.1.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "v0.12.1"
+linkTitle: "v0.12.1"
+weight: 20
+description: >
+---
+
+{{% pageinfo %}}
+[FRRouting](https://frrouting.org/) is currently not supported for MetalLB.
+{{% /pageinfo %}}
+
+### Configuring MetalLB in EKS Anywhere package spec
+
+| Parameter | Description | Default |
+|---|---|---|
+|**General**|||
+| address-pools[] | List of **address-pool** objects. Address-pools list the IP addresses that MetalLB is allowed to allocate, along with settings for how to advertise those addresses over BGP once assigned. You can have as many address pools as you want<br/>Example:<br/><pre>address-pools:<br/>  - name: default<br/>    protocol: bgp<br/>    addresses:<br/>      - 10.220.0.90/30| None |
+| peers[] | List of **peer** objects. The peers list tells MetalLB what BGP routers to connect too. There is one entry for each router you want to peer with. <br/>Example:<br/><pre>peers:<br/>  - peer-address: 10.220.0.2<br/>    peer-asn: 65000<br/>    my-asn: 65002</pre> | |
+| bgp-communities | (optional) BGP community aliases. Instead of using hard to read BGP community numbers in address pool advertisement configurations, you can define alias names here and use those elsewhere in the configuration.<br/>Example:<br/><pre>bgp-communities:<br/>  no-export: 65535:65281</pre>||
```

Comment:
```suggestion
| bgp-communities | (optional) BGP community aliases. Instead of using hard-to-read BGP community numbers in address pool advertisement configurations, you can define alias names here and use those elsewhere in the configuration.<br/>Example:<br/><pre>bgp-communities:<br/>  no-export: 65535:65281</pre>||
```